addons/l10n_cl_website_sale/views/templates.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="type_document_info" name="Chilean partner Type Document">
+        <div t-attf-class="mb-3 #{error.get('l10n_cl_sii_taxpayer_type') and '' or ''} col-lg-6 mb-0">
+            <div class="row">
+                <legend class="col-form-label col-6">Type Document</legend>
+                <div id="div_l10n_cl_sii_taxpayer_type"  class="col-sm-12">
+                    <div class="form-check form-check-inline">
+                        <input class="form-check-input" type="radio" name="l10n_cl_sii_taxpayer_type" id="radio_ticket"
+                               value="3"
+                               t-att-checked="'l10n_cl_sii_taxpayer_type' in checkout and checkout['l10n_cl_sii_taxpayer_type'] and checkout['l10n_cl_sii_taxpayer_type'] == '3'"/>
+                        <label class="form-check-label" for="ticket">Ticket</label>
+                    </div>
+                    <div class="form-check form-check-inline">
+                        <input class="form-check-input" type="radio" name="l10n_cl_sii_taxpayer_type"
+                               id="radio_invoice"
+                               value="1"
+                               t-att-checked="'l10n_cl_sii_taxpayer_type' in checkout and checkout['l10n_cl_sii_taxpayer_type'] and checkout['l10n_cl_sii_taxpayer_type'] == '1'"/>
+                        <label class="form-check-label" for="invoice">Invoice</label>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div id="div_l10n_cl_activity_description"
+             t-attf-class="mb-3 #{error.get('l10n_cl_activity_description') and 'o_has_error' or ''} col-lg-6"
+             t-att-style="( not 'l10n_cl_sii_taxpayer_type' in checkout or checkout['l10n_cl_sii_taxpayer_type'] in ['3', '', False] ) and 'display: none'">
+            <label class="col-form-label" for="l10n_cl_activity_description">Activity Description</label>
+            <input id="l10n_cl_activity_description" type="text" name="l10n_cl_activity_description"
+                   t-attf-class="form-control #{error.get('l10n_cl_activity_description') and 'is-invalid' or ''}"
+                   t-att-value="'l10n_cl_activity_description' in checkout and checkout['l10n_cl_activity_description']"/>
+        </div>
+    </template>
+
+    <template id="address_b2b" inherit_id="website_sale.address_b2b">
```

Comment:
@hea-bmya We might inherit from the base view immediately, but use the new _display_partner_b2b_fields method to inherit it so the field is always visible.